### PR TITLE
Update Devportal REST API Docs

### PR DESCRIPTION
**Pull request by msm1992**
## Purpose
This PR updates devportal REST API docs.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

**Review by Mariangela — APPROVED** [review]

**Comment by Mariangela** [comment]
@msm1992 , please get this technically reviewed before it is merged.

**Comment by Mariangela** [comment]
@msm1992 , please share the PR ID here once you add these changes to the carbon-apimgt repo.

File: en/docs/reference/product-apis/devportal-apis/devportal-v2/devportal-v2.yaml [hunk 1]
```diff
@@ -147,7 +147,14 @@ paths:
             "provider:wso2*" will match an API if the provider of the API starts with "wso2".
 
             Supported attribute modifiers are [**version, context, status,
-            description, subcontext, doc, provider, tag**]
+            description, subcontext, doc, provider, tag, label, api-category**]
```
Comment (malinthaprasan) [inline en/docs/reference/product-apis/devportal-apis/devportal-v2/devportal-v2.yaml:150]: only description, doc, provider, tag are supported

**Review by malinthaprasan — APPROVED** [review]

File: en/docs/reference/product-apis/devportal-apis/devportal-v2/devportal-v2.yaml (hunk 1 above)
Comment (msm1992) [reply en/docs/reference/product-apis/devportal-apis/devportal-v2/devportal-v2.yaml]: fixed with https://github.com/wso2/docs-apim/pull/3262/commits/da908d4c3376cf1d98b1e16c66cdfcb90fb96fa4

**Pull request merged**